xml/Microsoft.VisualC.StlClr.Generic/ContainerRandomAccessIterator`1.xml

```diff
@@ -560,10 +560,12 @@
       </ReturnValue>
       <Parameters>
         <Parameter Name="" Type="Microsoft.VisualC.StlClr.Generic.ContainerRandomAccessIterator&lt;TValue&gt;" RefType="ref" />
+        <Parameter Name="_Left" Type="System.Int32" />
```

Comment:
this change is wrong, please undo. The one with the _Left param is on line 598 and the reason that the param description is missing is because of an engineering bug that is not showing the description for empty param names.